frb_example/with_flutter/lib/main.dart

```diff
@@ -48,3 +49,13 @@ class _MyAppState extends State<MyApp> {
     if (mounted) setState(() => exampleText = receivedText);
   }
 }
+
+Future<void> _callExampleFfiChrono() async {
```

Comment:
Indeed we do not need to add tests in the with_flutter example - adding test into the pure_dart is enough.

Maybe move it to pure_dart?

Reply:
resolved in e32e967db05144d7c44a64e01f2c132db97dc96c

Reply:
on the Dart side there's not much, can we just skip it ?